src/logicblocks/event/processing/broker/broker/in_memory.py

```diff
@@ -0,0 +1,62 @@
+from datetime import timedelta
+
+from ....store import InMemoryEventStorageAdapter
+from ..locks import InMemoryLockManager
+from ..nodes import InMemoryNodeStateStore
+from ..sources import (
+    InMemoryEventStoreEventSourceFactory,
+)
+from ..subscribers import (
+    InMemoryEventSubscriberStateStore,
+)
+from ..subscriptions import InMemoryEventSubscriptionStateStore
+from .broker import EventBroker
+from .broker_builder import EventBrokerBuilder, PrepareResults
+
+
+class _InMemoryEventBrokerBuilder(EventBrokerBuilder):
```

Comment:
I don't like the `_` prefix. If a consumer of the package wanted to use this for some reason, I think that would be ok. It's not exported as one of the exports of the `logicblocks.event.processing.broker` package anyway so it's already clear it's not really a part of the public API.

Reply:
OK!